src/main/java/com/sprint/mission/discodeit/service/MessageService.java

```diff
@@ -0,0 +1,25 @@
+package com.sprint.mission.discodeit.service;
+
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface MessageService {
+    public void creat(Message message);
```

Comment:
채팅을 보낼 때 Message 객체를 인자로 넘기기보다 content, channelId, userId를 인자로 넘겨서 서비스단 내에서 Message 객체를 생성하는 방향은 어떨까요?  
```suggestion
    public void creat(String content, UUID userId, UUID channelId);
```